src/main/java/io/appium/java_client/AppiumDriver.java

```diff
@@ -228,6 +228,14 @@ private static CommandInfo deleteC(String url) {
         return super.findElementsById(id);
     }
 
+    @Override public T findElement(String by, String using) {
```

Comment:
It is not necessary to add _@Override public T findElement(String by, String using)_ 
but it is neccessary to 
 _@Override public List<T> findElements(String by, String using)_ (please pay attention)